### PR TITLE
Walt java

### PR DESCRIPTION
**Pull request by lmdgr34**
Login View

### CODE REVIEW

**Comment by lmdgr34** [comment]
Mood View and controllers part 1

File: src/app/MoodViewController.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import javafx.scene.control.Slider;
+
+/**
+ * FXML Controller class
+ *
+ * @author waltereckard
+ */
+public class MoodViewController implements Initializable {
+
+    //mood slider for excited
```
Comment (redjen) [inline src/app/MoodViewController.java:16]: The variable names need to match the associated fx:id, like moodAngry etc.

**Review by redjen — CHANGES_REQUESTED** [review]
Looking good, the only thing that needs to change are the variable names I noted because Java needs to have that in order to work. Alyson's working on the Account classes and might have changes for those two.

**Review by redjen — CHANGES_REQUESTED** [review]
This looks good but failed to launch for me with this: "Caused by: java.lang.IllegalArgumentException: Can not set javafx.scene.control.PasswordField field app.LoginController.password to javafx.scene.control.TextField"

Can you change the document loaded to the login view in App.java, then take a look at this? (we need that anyway) I think it's because the password field is PasswordFIeld in the controller but TextField in the view. You'll actually need to run the app to get the exception and logs. This isn't something that the compiler will catch, so it won't get flagged in NetBeans.

Once that's fixed this should be good to go.

**Comment by lmdgr34** [comment]
changes made

**Comment by redjen** [comment]
It complained about one more thing. :( The controller needs to have the handler for the button since it's defined in the view.

    @FXML
    public void loginButtonHandler(ActionEvent event) {

    }

**Review by redjen — APPROVED** [review]
The next task I have depends on this, so I'm just going to approve and merge it so I can get started. It's a little change so I'll just make it.

**Pull request merged**